README.md

```diff
@@ -317,20 +317,74 @@ sct_run_batch -jobs 32 -path-data ~/path-to-dataset/epi-stanford/ \
 ~~~
 
 
-## 8. Active learning procedure (TODO)
+## 8. Transfer learning from contrast-agnostic model
 
-> **Note**
-> This section is still a work in progress.
+This section provides instructions on how to train nnUNet models with pre-trained weights of the contrast-agnostic model. Ideal use cases include using the 
+pretrained weights for training/finetuning on any spinal-cord-related segmentation task (e.g. lesions, rootlets, etc.). 
+
+### Step 1: Download the pretrained weights
+
+Download the pretrained weights from the [latest release](https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases) of the 
+contrast-agnostic model. This refers to the `.zip` file of the format `model_contrast_agnostic_<date>_nnunet_compatible.zip`. 
+
+> [!WARNING]  
+> Only download the model with the `nnunet_compatible` suffix. If a release does not have this suffix, then that model weights are not directly 
+compatible with nnUNet.
+
+
+### Step 2: Create a new plans file
+
+1. Create a copy of the original `nnUNetPlans.json` file (found under `$nnUNet_preprocessed/<dataset_name_or_id>`) and 
```

Comment:
What do you think about this? To make explicit when to modify the `nnUNetPlans.json` file

```suggestion
1. After running `nnUNetv2_plan_and_preprocess` (and before running `nnUNetv2_predict`), create a copy of the original `nnUNetPlans.json` file (found under `$nnUNet_preprocessed/<dataset_name_or_id>`) and 
```

Reply:
looks good, more precise

Reply:
added in commit https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/129/commits/30f763dfd712e25dcd4db96e4e5c84d7f8336429

Reply:
thanks 👍🏻